common/components/common/projects/EditProjectForm.jsx

```diff
@@ -174,6 +174,7 @@ class EditProjectForm extends React.PureComponent<Props,State> {
             <option value="Kirkland, WA">Kirkland, WA</option>
             <option value="Bellevue, WA">Bellevue, WA</option>
             <option value="Seattle, WA">Seattle, WA</option>
+            <option value="Other">Other</option>
```

Comment:
We don't really want to show 'Other' as the location on the About Project page, so could you add that bit to AboutProjectController.jsx to hide the text if it's Other?